outboundSender.go

```diff
@@ -633,7 +635,7 @@ func (obs *CaduceusOutboundSender) queueOverflow() {
 		errorLog = logging.Error(obs.logger)
 	)
 
-	obs.queueEmpty = false
+	// obs.queueEmpty = false
```

Comment:
Is the commented out part intentional (should it be removed)?

Reply:
I can remove it, it's being set a little earlier